const_linux.go

```diff
@@ -0,0 +1,4 @@
+package main
+
+// MaxCommandBytes is the maximum number of bytes supported by exec
+const MaxCommandBytes = 2097152
```

Comment:
We can control the environment exposed to the process, so we should be able to account for those bytes.

Using 32k would potentially mean that linters run up to 4x slower on later projects (8x slower on OSX). I think that would really be unfortunate. Could we try try with something like 131072 ? We could use 130000 so there's an extra 1k buffer.  32k on windows and "other" platforms.